### PR TITLE
Previews status table

### PR DESCRIPTION
**Pull request by SeanFeldman**
_Replaces https://github.com/Particular/docs.particular.net/pull/4854_

### CODE REVIEW

File: previews/index.md [hunk 1]
```diff
@@ -10,3 +10,9 @@ Before deciding to launch a Preview, a research process is undertaken to determi
 Previews typically run for some period of time and their success is determined by customer adoption. If successful, it will translate into the adoption into the Platform. Previews that are insufficiently used in production will be converted into a community project.
 
 During a Preview, user feedback is central, so the more feedback is received, the better the end result. 
+
+## Previews
```
Comment (sergioc) [inline previews/index.md:14]:
```suggestion
## Current Previews
```

File: previews/index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Platform previews
```
Comment (sergioc) [inline previews/index.md:2]:
```suggestion
title: Previews
```

File: previews/index.md (hunk 1 above)
Comment (SeanFeldman) [reply previews/index.md:14]: The table of previews already has status and the title will repeat the status.
We could change the title to a 'List of previews' if that helps.

Comment (SeanFeldman) [reply previews/index.md:14]: I'm going to resolve this comment. If this is something that is still a problem, let's review it as a squad (via inbox).

**Comment by SeanFeldman** [comment]
Will be handled in the Azure Functions PR #4879

**Pull request closed without merging**